test/1_stdlib/Runtime.swift.gyb

```diff
@@ -995,7 +995,7 @@ Reflection.test("ObjectIdentifier/CustomDebugStringConvertible") {
   expectEqual(String(reflecting: oi1), String(reflecting: oi1))
   expectNotEqual(String(reflecting: oi1), String(reflecting: oi2))
 
-  let p1 = UnsafePointer<Void>(bitPattern: UInt(oi1))!
+  let p1 = UnsafePointer<Void>(bitPattern: UInt(bitPattern: oi1))!
   expectPrinted("ObjectIdentifier(\(p1))", oi1)
```

Comment:
Could you add a test for `Int(bitPattern:)` as well?  Seems like that API is not covered by any other tests.